ios/FluentUI.Demo/FluentUI.Demo/DemoAppearanceController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+import SwiftUI
+
+/// Wrapper class to allow presenting of `DemoAppearanceView` from a UIKit host.
+class DemoAppearanceController: UIHostingController<DemoAppearanceView> {
+    init() {
+        let configuration = DemoAppearanceView.Configuration()
+        self.configuration = configuration
+
+        super.init(rootView: DemoAppearanceView(configuration: configuration))
+    }
+
+    @MainActor @objc required dynamic init?(coder aDecoder: NSCoder) {
```

Comment:
Why do we need @MainActor / @objc / dynamic?

Reply:
Ha, no idea. Xcode wrote this line of code for me 🥇 

(I'll remove the things that I don't understand and that don't actually seem necessary.)